### PR TITLE
Update calicoctl docs for get/delete

### PR DESCRIPTION
**Pull request by tomastigera**
This is to match https://github.com/projectcalico/calicoctl/pull/1998

## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit f6050b8a2a8749464f06b35c7a47420b341ecd93

https://deploy-preview-2593--calico.netlify.com

File: master/reference/calicoctl/commands/get.md [hunk 1]
```diff
@@ -23,16 +23,17 @@ command.
...
                 --filename=<FILENAME>)
                 [--output=<OUTPUT>] [--config=<CONFIG>] [--namespace=<NS>] [--all-namespaces]
 
 Examples:
   # List all policy in default output format.
   calicoctl get policy
 
-  # List a specific policy in YAML format
-  calicoctl get -o yaml policy my-policy-1
+  # List a specific policies in YAML format
```
Comment (caseydavenport) [inline master/reference/calicoctl/commands/get.md]: Probably should be "List specific policies" rather than "List a specific policies"

Comment (tomastigera) [reply master/reference/calicoctl/commands/get.md]: Damn, so it is also wrong in the calicoctl code as I copy-pasted it here :-/

Comment (tomastigera) [reply master/reference/calicoctl/commands/get.md]: done

**Pull request merged**